plenum/server/validator_info_tool.py

```diff
@@ -165,8 +195,258 @@ def __unreachable_list(self):
+            replica_stat["Last_ordered_3PC"] = self._prepare_for_json(replica.last_ordered_3pc)
+            stashed_txns = {}
+            stashed_txns["Stashed_checkoints"] = self._prepare_for_json(len(replica.stashedRecvdCheckpoints))
+            if replica.prePreparesPendingPrevPP:
+                stashed_txns["Min_stashed_PrePrepare"] = self._prepare_for_json(
+                    replica.prePreparesPendingPrevPP.itervalues[-1])
+            replica_stat["Stashed_txns"] = stashed_txns
+            res[replica.name] = self._prepare_for_json(replica_stat)
+        return res
+
+    @property
+    @none_on_fail
+    def __node_info(self):
+        ledger_statuses = {}
+        waiting_cp = {}
+        num_txns_in_catchup = {}
+        last_txn_3PC_keys = {}
+        root_hashes = {}
+        uncommited_root_hashes = {}
+        uncommited_txns = {}
+        for idx, linfo in self._node.ledgerManager.ledgerRegistry.items():
+            ledger_statuses[idx] = self._prepare_for_json(linfo.state.name)
+            waiting_cp[idx] = self._prepare_for_json(linfo.catchUpTill)
+            num_txns_in_catchup[idx] = self._prepare_for_json(linfo.num_txns_caught_up)
+            last_txn_3PC_keys[idx] = self._prepare_for_json(linfo.last_txn_3PC_key)
+            if linfo.ledger.uncommittedRootHash:
+                uncommited_root_hashes[idx] = self._prepare_for_json(base58.b58encode(linfo.ledger.uncommittedRootHash))
+            uncommited_txns[idx] = [self._prepare_for_json(txn) for txn in linfo.ledger.uncommittedTxns]
+            if linfo.ledger.tree.root_hash:
+                root_hashes[idx] = self._prepare_for_json(base58.b58encode(linfo.ledger.tree.root_hash))
+
+        replicas_status = self._prepare_for_json(self.__replicas_status)
+        ic_queue = {}
+        for view_no, queue in self._node.view_changer.instanceChanges.items():
+            ics = {}
+            ics["Vouters"] = self._prepare_for_json(list(queue.voters))
+            ics["Message"] = self._prepare_for_json(queue.msg)
+            ic_queue[view_no] = self._prepare_for_json(ics)
+        metrics = self._node.monitor.metrics()
+        if metrics:
+            metrics = [self._prepare_for_json(m) for m in metrics]
+        return {
+            "Node info": {
+                "Name": self._prepare_for_json(self._node.name),
+                "Last N pool ledger txns": "",
+                "Mode": self._prepare_for_json(self._node.mode.name),
+                "Metrics": self._prepare_for_json(metrics),
+                "Root_hashes": self._prepare_for_json(root_hashes),
+                "Uncommitted_root_hashes": self._prepare_for_json(uncommited_root_hashes),
+                "Uncommitted_txns": self._prepare_for_json(uncommited_txns),
+                "View_change_status": {
+                    "View_No": self._prepare_for_json(self._node.viewNo),
+                    "VC_in_progress": self._prepare_for_json(self._node.view_changer.view_change_in_progress),
```

Comment:
We need to provide info from what view have we started the view change:
"View Change from viewNo X to viewNoY is in progress"